daemon/networkdriver/portallocator/portallocator_test.go

```diff
@@ -5,6 +5,11 @@ import (
 	"testing"
 )
 
+func init() {
+	beginPortRange = DefaultPortRangeStart
+	endPortRange = DefaultPortRangeStart + 500
```

Comment:
Agree with @rhatdan, we might as well test the entire range for consistency, but it's not a blocker.